### DISCUSSION TITLE
sun t2000 or HP dl360 g9

### DISCUSSION THREAD

**Discussion by Andygithub68** [roflcoopter/viseron#749] [General]
I have 2 different spare servers that I was hoping to use Viseron with/on. I have 1, or 2, spare sun t2000 servers (sparc cpu, but otherwise unkown config @ this time), and a spare HP dl360 G9 (1 cpu, prolly 16 gig ram (but will upgrade if to be used), but basically unknown config @ this time).  

I already have an NVR for camera purposes, but am really not pleased with it, which is bnc cabling.  I'd like to go with 4MP or higher IP cams, but the cams will be way after I get server and software working. It's a work in progress kind of thing, but I'd rather sooner, than later.

My question is, for now anyway, is which server would be better and with which OS? I also have like a 16 gig, and 24 gig Tesla card than can be used if needed.  I'm old, and old school, so I kinda over-engineer stuff, but if I like the way it works, then nothing will change for a long time. :P  Sun T2000 or HP dl360G9? Winblows or linux/unix? (I'm better with unix, than linux, which is better than winblows), But I can handle all 3. 📦 
Thanks for your help/guidance/criticisms.

Andy

**Comment by bsyomov** [comment]
Linux on HP dl360 G9.

**Comment by roflcoopter** [comment]
Windows is not supported, so definitely linux.

As for which machine to run on, you probably know better than me!
If you are going to use the GPU most of the computing can be offloaded to it which makes the CPU less relevant

**Reply by Andygithub68** [reply]
So Linux it is! Now which Flavor and version works best/easiest?  I used to be really good @ Solaris and BSD Unix, but that was some time ago, and I remember that I used the like Red Hat Linux, but again it's been awhile.  I was much better with command line than the gui they had back then, but I understand it's gotten better.

Since I'll be using the HP dl360 Gen9, which is 1u, I won't be able to use the 2 different Tesla cards I have (they are 2 bay cards, and I only have 1 bay avail.), but I'll take a look and see what my options are for that.

Are there any tips/tricks/hints that anyone can suggest?  I'll prolly start working on getting server in rack, and whatnot, this weekend.  From what I've seen in the docs and such, it seems pretty straightforward and should be relatively easy.

Thanks for your help and suggestions.

Andy

**Reply by bsyomov** [reply]
Ubuntu server is a good choice. You don't need any GUI at all. Minimal install + nvidia drivers + docker + probably http reverse proxy.

**Comment by thetravellor** [comment]
I would recommend Debian Linux for older or obscure systems like your Sun
machine. It is highly stable and is very broadly supported in the community.
